examples/llava/clip.h

```diff
@@ -45,7 +45,7 @@ struct clip_context_params {
 };
 
 // deprecated, use clip_init
-CLIP_API struct clip_ctx * clip_model_load(const char * fname, int verbosity);
+CLIP_API struct clip_ctx * clip_model_load(const char * fname, const int verbosity=1, const bool use_gpu=true);
```

Comment:
Do not change the signature of this function, many downstream binding will break.

Instead, use `clip_init` and set `user_gpu` via `clip_context_params`

Reply:
Also the `const int verbosity=1` will fail to compile in C, some users use this header file in C projects.